www/docs/client/links/httpBatchStreamLink.md

```diff
@@ -15,6 +15,10 @@ We have prefixed this as `unstable_` as it's a new API, but you're safe to use i
 
 > All usage and options are identical to [`httpBatchLink`](./httpBatchLink.md).
 
+:::note
+If you require the ability to change/set response headers (which includes cookies) from within your procedures, make sure to use `httpBatchLink` instead! This is due to the fact that `unstable_httpBatchStreamLink` does not support setting headers once the stream has begun. [Read more](https://trpc.io/docs/client/links/httpBatchLink).
```

Comment:
nit, ubt this sort of makes it sound like it's a limitation on the link itself, but it's the opposite - streaming in general doesn't support setting headers after the stream has begun